direct/nn/openvino/openvino_model.py

```diff
@@ -0,0 +1,58 @@
+from openvino_extensions import get_extensions_path
+from openvino.inference_engine import IECore
+
+import torch
+import torch.nn as nn
+
+import subprocess
+import sys
+import os
+
+
+class OpenVINOModel(nn.Module):
+    def __init__(self, model):
+        super().__init__()
+        self.model = model
+
+    def forward(self, input_image, masked_kspace, sampling_mask, sensitivity_map, loglikelihood_scaling=None):
+        input_map = {
+            "input_image": input_image,
+            "masked_kspace": masked_kspace,
+            "sampling_mask": sampling_mask,
+            "sensitivity_map": sensitivity_map,
+        }
+
+        origin_forward = self.model.forward
+        self.model.forward = lambda x: origin_forward(
+            input_image, masked_kspace=masked_kspace, sampling_mask=sampling_mask, sensitivity_map=sensitivity_map
+        )
+
+        torch.onnx.export(
+            self.model,
+            input_map,
+            "model.onnx",
+            opset_version=11,
+            enable_onnx_checker=False,
+            input_names=["input_image", "masked_kspace", "sampling_mask", "sensitivity_map"],
+        )
```

Comment:
@likholat, please check this once again:
```
    def forward(self, input_image, masked_kspace, sampling_mask, sensitivity_map, loglikelihood_scaling=None):

        torch.onnx.export(
            self.model,
            [input_image, masked_kspace, sampling_mask, sensitivity_map],
            "model.onnx",
            opset_version=11,
            enable_onnx_checker=False,
            input_names=["input_image", "masked_kspace", "sampling_mask", "sensitivity_map"],
        )
```

without lambda that should work

Reply:
`E           TypeError: forward() missing 2 required positional arguments: 'masked_kspace' and 'sampling_mask'`